axis.yaml

```diff
@@ -1,29 +1,27 @@
-BUILD_IMAGE:
-  - "rapidsai/citestwheel:cuda-devel-11.8.0-ubuntu18.04"
-  - "rapidsai/citestwheel:cuda-devel-12.0.1-ubuntu18.04"
-  - "rapidsai/citestwheel:cuda-devel-11.8.0-ubuntu20.04"
-  - "rapidsai/citestwheel:cuda-devel-12.0.1-ubuntu20.04"
-  - "rapidsai/manylinux_v2_2014:cuda-devel-11.8.0-centos7"
-  - "rapidsai/manylinux_v2_2014:cuda-devel-12.0.1-centos7"
-  - "rapidsai/manylinux_v2_2_31:cuda-devel-11.8.0-ubuntu20.04"
-  - "rapidsai/manylinux_v2_2_31:cuda-devel-12.0.1-ubuntu20.04"
-RUNNER_ARCH:
-  - "amd64"
-  - "arm64"
+CUDA_VER:
+  - "11.8.0"
+  - "12.0.1"
+PYTHON_VER:
+  - "3.9.16"
+  - "3.10.9"
```

Comment:
To follow the conda image conventions, are we able to omit the patch version here?

It looks like the only place these values are used are in the `pyenv global ${PYTHON_VER}` command in the Dockerfiles.

Does that command work with just a major minor version?

Reply:
Good question, let me try without and see what happens

Reply:
Okay, this worked just fine